app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
```

Comment:
If you match the column name here (please do so), Rails will do some magic 🙌 .

Then, you can also remove the prefix `difficulty` from the answer options.